### ISSUE TITLE
Color contrast issues in the demo

### ISSUE THREAD

**Issue by mgifford** [openemr/openemr#7904]
### Describe the bug

Looking at the demos

https://demo.openemr.io/openemr/interface/main/tabs/main.php
https://demo.openemr.io/a/openemr/interface/main/tabs/main.php
https://demo.openemr.io/b/openemr/interface/main/tabs/main.php

![Screenshot of low contrast menu 1](https://github.com/user-attachments/assets/aed0dfb0-dcc0-4b20-b404-c306d8b32b88)
![Screenshot of low contrast menu 2](https://github.com/user-attachments/assets/408c76ba-1eb2-4e63-b027-1b6e300c1745)
![Screenshot of low contrast menu 3](https://github.com/user-attachments/assets/2fd9f438-e9da-4cbf-bd92-976cb4e05808)

None of those have good color contrast.  They will all fail WCAG 2.0.

### To Reproduce

Open up - https://www.open-emr.org/demo/
Login as admin

I suggest trying https://accessibilityinsights.io for this

### Expected behavior

The colors should meet WCAG color contrast requirements.

Another great tool to try is https://contrast-finder.tanaguru.com

### Client configuration

- Browser: Edge
- OpenEMR version: OpenEMR 7.0.2
- Operating system: Mac (well my desktop, not your server)
